_installation/compiling-babelfish-from-source.md

```diff
@@ -6,261 +6,211 @@ nav_order: 2
 
 # Compiling Babelfish from source
 
-This section will walk you through the steps required to create a working Babelfish installation on Linux. 
+This section will walk you through the steps required to create a working Babelfish installation on an Ubuntu 20.04 Linux host. Please note that the steps may vary on other operating systems, but the overall process is roughly the same. 
 
 
 ## Downloading the source code
 
-Before building Babelfish, you will need to download the source code.  Babelfish source code is separated into two repositories that contain:
+Before building Babelfish, you will need to download the source code.  Babelfish source code is stored in the following repositories: FIXME-are_these_still_the_same_repo_names_?
```

Comment:
We should start using the release tarballs.

Reply:
What about including another file with "Compiling and Installing from tarballs"? Steps slightly vary, but it'll be more readable for users.  IMHO both methods should coexist.  cc/ @nasbyj